tests/src/test/java/alluxio/client/rest/S3ClientRestApiTest.java

```diff
@@ -105,8 +100,12 @@ public final class S3ClientRestApiTest extends RestApiTest {
   @ClassRule
   public static LocalAlluxioClusterResource sResource = new LocalAlluxioClusterResource.Builder()
       .setIncludeProxy(true)
-      .setProperty(PropertyKey.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, false)
-      .setProperty(PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.SIMPLE)
+      // TODO(czhu): Add test cases handling "Authorization" header w/ Alluxio Authorization
+      .setProperty(PropertyKey.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, true) // default
+      .setProperty(PropertyKey.SECURITY_AUTHENTICATION_TYPE,
+          AuthType.SIMPLE) // default, TestCaseOptions.defaults() sets the "Authorization" header
+      .setProperty(PropertyKey.S3_REST_AUTHENTICATION_ENABLED, // TODO(czhu) refactor this key name
+          false) // default, disables AWS "Authorization" header signature validation
```

Comment:
Enabling testing the S3 API w/ Authorization via the Alluxio Master ACLs.
- Authentication is still just taken at face value from the "Authorization" header